### ISSUE TITLE
test_5_shot.py

### ISSUE THREAD

**Issue by zhao123ying** [Yang-Bob/PMMs#27]
Hello,I have a question.Why merge on dim=2  not dim=1 in the def forward_5shot () function in test_5_shot.py?
support_feature_all = torch.cat([support_feature_all, support_feature], dim=2) #torch.Size([1, 256, 305, 61]) support_mask_all = torch.cat([support_mask_all, support_mask_temp], dim=2)#torch.Size([1, 1, 305, 61]

**Comment by Yang-Bob** [comment]
Hello, sorry to reply so late. 
As for 5-shot learning, the five support images are fed into the PMMs to estimate prototypes together. Specifically, those support images are merged into a bigger image. Thus, they are merged on dim=2 (width) or dim=3 (height) not dim=1(channel).
